src/encoded/tests/test_purge_item_type.py

```diff
@@ -1,31 +1,31 @@
 import pytest
 import time
 from dcicutils.qa_utils import notice_pytest_fixtures
-from .workbook_fixtures import app_settings, app, workbook
+# from .workbook_fixtures import app_settings, app, workbook
```

Comment:
`conftest.py` now has the support for this, and is loaded automatically. I ported @willronchetti's changes which added an `es_xxx` fixture for those things that would have been done by the workbook test version of app, etc.  I'll remove the commented-out stuff at some point but for now it's reminding me that this file recently changed implementation strategy.